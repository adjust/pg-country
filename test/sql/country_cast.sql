BEGIN;
CREATE EXTENSION country;
SELECT 'de'::country::int4;
SELECT 'de'::country::int2;
SELECT 57::int4::country;
SELECT 57::int2::country;
ROLLBACK;
