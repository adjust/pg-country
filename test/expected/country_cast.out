BEGIN;
CREATE EXTENSION country;
SELECT 'de'::country::int4;
 int4 
------
   57
(1 row)

SELECT 'de'::country::int2;
 int2 
------
   57
(1 row)

SELECT 57::int4::country;
 country 
---------
 de
(1 row)

SELECT 57::int2::country;
 country 
---------
 de
(1 row)

ROLLBACK;
